packages/reactive-element/src/decorators/base.ts

```diff
@@ -17,3 +17,10 @@ export type Constructor<T> = {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   new (...args: any[]): T;
 };
+
+export const descriptorDefaults = {
+  enumerable: true,
+  configurable: true,
+} as const;
+
+export const extendedReflect: typeof Reflect & {decorate?: unknown} = Reflect;
```

Comment:
I only see this accessed as `extendedReflect.decorate` coerced to a boolean, so could you just export that boolean?

```suggestion
export const reflectDecorateExists = 'decorate' in Reflect;
```